pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -104,3 +107,20 @@ def attach_model_logging_functions(self, model):
         for callback in self.trainer.callbacks:
             callback.log = model.log
             callback.log_dict = model.log_dict
+
+
+def _reorder_callbacks(callbacks: List[Callback]) -> List[Callback]:
```

Comment:
Any reason to have this outside the CallbackConnector class?

Reply:
because it doesn't depend on connectors state
staticmethod?

Reply:
do you want it to be static?

Reply:
yes! 👍

Reply:
done